src/index.tsx

```diff
@@ -12,7 +12,7 @@ const style = {
     transition: "background 0.25s ease-in-out",
     backgroundPosition: "center",
```

Comment:
@Djspaceg ping? Would you be able to make this change and we can release this improvement?